### ISSUE TITLE
Update tables using “$” to provide more context to screen readers

### ISSUE THREAD

**Issue by tbolt** [CMSgov/eAPD#2547]
Certain tables that have dollar-amount inputs have a fixed dollar-sign "$" in a table cell. This should be investigated and made accessible.

### This task is done when...
 - [ ] "$" in tables are provided with more context in text fields

**Comment by tbolt** [comment]
It appears this is not an issue as we are using the CMS Design System Masked Field component: https://design.cms.gov/components/masked-field/

The behavior of our application is the same as the design systems as well. Any issues with this should be addressed via an issue opened with the cmsds team

CC: @rohitpaul @nicholeweems
